entity-management-web/src/main/java/eu/europeana/entitymanagement/serialization/EMTextSerializer.java

```diff
@@ -0,0 +1,13 @@
+package eu.europeana.entitymanagement.serialization;
+
+import eu.europeana.corelib.edm.model.schemaorg.Text;
+import eu.europeana.corelib.edm.utils.TextSerializer;
+
+public class EMTextSerializer extends TextSerializer {
```

Comment:
is this class needed? Can we use the TextSerializer directly?

Reply:
Yes for the time being it is needed, since the TextSerializer constructor in not public. It needs to be changed in the corelib module and pushed to the maven repository so that we can use it properly.